java-manta-client/src/main/java/com/joyent/manta/client/crypto/MantaEncryptedObjectInputStream.java

```diff
@@ -51,7 +52,7 @@
     /**
      * Default buffer size to use when reading chunks of data from streams.
      */
-    private static final int DEFAULT_BUFFER_SIZE = 8192;
+    private static final int DEFAULT_BUFFER_SIZE = 512;
```

Comment:
I did some benchmarking and the CipherInputStream doesn't benefit from a greater buffer size.